omnigibson/robots/fetch.py

```diff
@@ -501,3 +498,17 @@ def arm_workspace_range(self):
         return {
             self.default_arm : [np.deg2rad(-45), np.deg2rad(45)]
         }
+    
+    @property
+    def eef_usd_path(self):
+        return {self.default_arm: os.path.join(gm.ASSET_PATH, "models/fetch/fetch/fetch_eef.usd")}
+
+    @property
+    def teleop_rotation_offset(self):
+        return {self.default_arm: euler2quat([0, np.pi / 2, np.pi])}
+    
+    def teleop_data_to_action(self, teleop_data: dict):
+        action = np.zeros(self.action_dim)
+        action[:2] = TwoWheelRobot.teleop_data_to_action(self, teleop_data)
```

Comment:
ideally this should be a bit more programmatic -- i.e.: infer the index ordering from `controller_order` and the length of each index from `controller.command_dim`

Reply:
Hmm I'm actually not sure about this. I think using `controller_order` and `command dim` will introduce unnecessary redundancies because index ordering is already specific to each robot?

Reply:
add action_control_idx to base robot

Reply:
done